contracts/base/ModuleManager.sol

```diff
@@ -275,4 +283,20 @@ abstract contract ModuleManager is SelfAuthorized, Executor, IModuleManager {
             moduleGuard := sload(slot)
         }
     }
+
+    /**
+     * @notice A hook that gets called after execution of {execTransactionFromModule*} methods.
+     * @param to Destination address of module transaction.
+     * @param value Ether value of module transaction.
+     * @param data Data payload of module transaction.
+     * @param operation Operation type of module transaction.
+     * @param success Boolean flag indicating if the call succeeded.
+     */
+    function _onAfterExecTransactionFromModule(
```

Comment:
I feel horrible because I suggested the name, but none of the functions in the Safe contracts use the `_` prefix. For naming consistency, I would suggest:

```suggestion
    function onAfterExecTransactionFromModule(
```